.github/workflows/conformance-eks-v1.11.yaml

```diff
@@ -259,6 +256,8 @@ jobs:
       - name: Enable Relay
         run: |
           cilium hubble enable ${{ steps.vars.outputs.hubble_enable_defaults }}
+          # NB: necessary to work against occassional flakes due to https://github.com/cilium/cilium-cli/issues/918
+          cilium status --wait
```

Comment:
Nit: That would be best placed in a separate commit so we can just revert when the time comes (my understanding is that it's just a workaround).

Reply:
Good point, will move to a separate commit.